test/e2e/test/elasticsearch/builder.go

```diff
@@ -205,13 +210,19 @@ func (b Builder) WithEmptyDirVolumes() Builder {
 	return b
 }
 
-func (b Builder) WithPersistentVolumes(volumeName string) Builder {
+func (b Builder) WithDefaultPersistentVolumes() Builder {
+	storageClass := defaultStorageClass
 	for i := range b.Elasticsearch.Spec.NodeSets {
-		name := volumeName
+		for _, existing := range b.Elasticsearch.Spec.NodeSets[i].VolumeClaimTemplates {
+			if existing.Name == defaultVolumeName {
+				goto next
```

Comment:
one way of avoiding the `goto` statement is to extract the code block within the for loop to its own function, in which case we can use `return`.
But I'm fine with `goto`, assembly is fun.

Reply:
All other `With.*` functions are contained, so I didn't want to break that :)